packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/users-tokens/users-tokens.html

```diff
@@ -0,0 +1,150 @@
+<div class="mt-2">
+    <div class="mt-4">
+        <cui-message-container
+            data-messages="$ctrl.messages"
+        ></cui-message-container>
+    </div>
+    <!--Title & desc-->
+    <h3 data-translate="pci_ai_dashboard_users_tokens_title"></h3>
+    <p data-translate="pci_ai_dashboard_users_tokens_description"></p>
+    <!--Users-->
+    <section>
+        <h4 data-translate="pci_ai_dashboard_users_tokens_users_title"></h4>
+        <div class="mb-3">
+            <span
+                data-translate="pci_ai_dashboard_users_tokens_users_description_1"
+            ></span>
+            <br />
+            <span
+                data-translate="pci_ai_dashboard_users_tokens_users_description_2"
+            ></span>
```

Comment:
**nitpick:** `<p>` would be a better choice if you want them in two different lines.

Reply:
Changed here: https://github.com/ovh/manager/pull/8443/commits/1e785e41932cdb7bbac7de9f5126103c8cb4d899